lib/parser.js

```diff
@@ -109,7 +110,9 @@ var valid = [
     'amaze',
     'classy',
     'maker',
-    'grows'
+    'grows',
+    'git',
+    'sit'
```

Comment:
do we want to keep a trailing comma here?

Reply:
Yeah, I agree. I was in the mindset of fixing everything and didn't consider to split all my changes.